examples/advanced/llm_hf/requirements.txt

```diff
@@ -1,8 +1,8 @@
 nvflare~=2.5.0rc
-torch
+torch==2.5.1
 datasets
 tensorboard
-transformers
-peft
-trl
+transformers==4.44.2
+peft==0.13.2
+trl==0.11.4
```

Comment:
what is the huggingface version?
are these restrictions come from huggingface lib?

Reply:
all three of them are from huggingface, the reported issue from QA is caused by HF's api updates on these libs, so it will be better to specify the version rather than updating the code